server/routes/convo.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+
+const auth = require("../middleware/auth");
+
+// Connect to conversation controller
+const convoController = require("../controllers/convo");
+
+/* POST route to create a conversation - POST CONVERSATION operation. */
```

Comment:
These comments are pretty unnecessary as the code explains itself in this instance, I would remove all the comments about the routes.